src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -309,6 +313,27 @@ func (cfg *Config) ApplyDefaults() {
 	}
 }
 
+func defaultRoundRobin() string {
+	localeTerritory, err := jibber_jabber.DetectTerritory()
+	if err != nil {
+		localeTerritory = "us"
+	}
+	log.Debugf("Locale territory: %v", localeTerritory)
+	return defaultRoundRobinForTerritory(strings.ToLower(localeTerritory))
+}
+
+// defaultDataCenter customizes the default data center depending on the user's locale.
+func defaultRoundRobinForTerritory(localeTerritory string) string {
+	datacenter := ""
+	if localeTerritory == "cn" {
```

Comment:
I saw in the tests you used uppercase and here you are only testing for lowercase. Is there any possibility that the input is uppercase?